### PR TITLE
Move intro slide for advanced into their own file

### PR DESCRIPTION
**Pull request by topepo**

### CODE REVIEW

File: slides/advanced-01-introduction.qmd [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+set.seed(4028)
+hotel_split <- initial_split(hotel_rates, strata = avg_price_per_room)
+
+hotel_train <- training(hotel_split)
+hotel_test <- testing(hotel_split)
+```
+
+
+
+## Your turn {transition="slide-in"}
```
Comment (topepo) [inline slides/advanced-01-introduction.qmd:192]: moved the warm-up here

File: slides/advanced-03-tuning-hyperparameters.qmd [hunk 2]
```diff
@@ -69,10 +69,10 @@ reg_metrics <- metric_set(mae, rsq)
 #| label: data-import
 set.seed(295)
 hotel_rates <- 
-  hotel_rates %>% 
+  data_hotel_rates() %>% 
   sample_n(5000) %>% 
   arrange(arrival_date) %>% 
-  select(-arrival_date_num, -arrival_date) %>%
```
Comment (topepo) [inline slides/advanced-03-tuning-hyperparameters.qmd:75]: fixed a few small data issues (this should have been retained)

File: slides/advanced-01-introduction.qmd [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+#| out-width: "20%"
+#| echo: false
+
+knitr::include_graphics("images/initial-split.svg")
+```
+
+
+## Data Spending `r hexes("rsample")`
+
+Let's split the data into a training set (75%) and testing set (25%):
```
Comment (EmilHvitfeldt) [inline slides/advanced-01-introduction.qmd:183]: Do you wanna comment on that you are doing stratification?

**Review by EmilHvitfeldt — APPROVED** [review]
looking good!

**Pull request merged**